File: zerog_scotlandyard/Detective.java
package zerog_scotlandyard;
import java.util.*;
import static zerog_scotlandyard.TransportType.*;

public class Detective extends Player {
	public Detective(String name, int id){
		super(name, id);
		tickets.put(taxi, 10);
		tickets.put(bus, 8);
		tickets.put(underground, 4);
	}
}

File: tests/Tests/Detective.cs
using System.Collections;

public class Detective :  Player {
	public Detective (string name, int id) : base(name,id){
		_tickets.Add(TransportType.taxi, 10);
		_tickets.Add(TransportType.bus, 8);
		_tickets.Add(TransportType.underground, 4);
	}
}
